mdn/developer-portal
# Implement redesigned tabbed panel component

This changeset addresses issues around the tabbed panel component by implementing a cleaner, simpler design that scales better. Previously there were issues with margins, fixed heights and a generally cluttered look, whereas now information is easier to read and the panel can be used for a variety of things (seeing as only adding one item essentially displays the section as a banner).

There are also bug fixes and general improvements to the component throughout. *Note:* the design of the mobile version has remained unchanged.

Before:
![Screen Shot 2019-10-09 at 6 09 39 PM](https://user-images.githubusercontent.com/1469007/66504600-5eb4b680-eac1-11e9-93cf-8932279bbfaf.png)

After:
![Screen Shot 2019-10-09 at 6 09 28 PM](https://user-images.githubusercontent.com/1469007/66504606-62483d80-eac1-11e9-80f0-25008957a0a8.png)

## Key changes

- Implement redesigned component and clean up related files.
- Remove unneeded tabbed panel heading field from topics.
- Fix spacing issues within pattern header and featured card containers.
- Fix a couple of small issues with the mobile design.

## How to test

- Add one tabbed panel item to a topic. Observe that the section looks correct.
- Continue to add more items, until the max of 3 and test responsively.
- Ensure both internal and external links work as expected.
